lib/ansible/modules/identity/ipa/ipa_dnsrecord.py

```diff
@@ -99,6 +99,16 @@
     record_type: 'AAAA'
     record_ip: '::1'
 
+# Ensure a PTR record is present
+- ipa_dnsrecord:
+    ipa_host: spider.example.com
+    ipa_pass: Passw0rd!
+    state: present
+    zone_name: 2.168.192.in-addr.arpa
+    record_name: 5
+    record_type: 'PTR'
+    record_ip: 'internal.ipa.example.com'
```

Comment:
Ah! I am OK with this approach as well. Go ahead with changes - `record_value` replacing `record_ip`.